frontend/src/components/RosalutionHeader.vue

```diff
@@ -105,7 +105,6 @@ header .content {
   display: flex;
   flex: 1 1 auto;
   align-items: center;
-  gap: 12px;
```

Comment:
This gap needs to be re-inserted so that the spacing between flexbox objects work as they did initially. https://github.com/uab-cgds-worthey/rosalution/pull/64/files#r1167088593  Keeping this spacing will prevent us from having to manually add additional margins for the spacing it accounted for in each of the headers that implement the slot.

Reply:
Reintroducing the mentioned code results in a spacing issue concerning the icons, where their position becomes unmodifiable. Is there a method you're aware of that could achieve both objectives?
![image](https://user-images.githubusercontent.com/4248757/232152999-d2efbeea-63ec-4f14-82ed-42dfbd4d7850.png)

Reply:
Yes; I encourage you to invest time on css to resolve this.

Reply:
I appreciate your suggestion to invest more time learning CSS to resolve the issue. However, I'm seeking more specific guidance or resources to help me understand and address the problem more effectively. While I'm eager to learn and improve my skills, there may be more efficient ways to assist me than simply directing me to explore alone with little guidance. Please provide more targeted advice or examples to help me grasp the concept better. I appreciate your understanding.

Reply:
Hi Alex. Ange doesn't have the time to write everything up for you. In the link provided previously, Ange already directed you to what seems to be an excellent resource for info on this very topic.... You need to take the initiative to find things and learn; that is the expectation on you at this point. Please do the reading as suggested. in addition; a quick search of the web turned up lots of reading on CSS. You have all the info you need for this task. Please resolve asap.

Reply:
<img width="423" alt="image" src="https://user-images.githubusercontent.com/1209059/232650670-0003fc65-f9aa-48fb-9065-43f20837db48.png">

Confirmed gap was rei-inserted and spacing was migrated to be within the headers.